src/pages/mesh/advanced/caching/index.md

```diff
@@ -178,3 +191,37 @@ Cache-control header values in your mesh configuration file take precedence over
   }
 }
 ```
+
+## Use your own CDN
+
+To disable native caching in API Mesh and use your own CDN, ensure that your `responseConfig` contains `"cache": false` to avoid double caching.
+
+While we recommend using the [native API Mesh caching](#api-mesh-native-caching) functionality, you can also provide your own content delivery network (CDN), such as Fastly. Refer to the [Fastly caching example](./fastly.md) for more information.
+
+<InlineAlert variant="info" slots="text"/>
+
+When using your own CDN, you must invalidate the cache after modifying a mesh configuration, or you will receive stale information. There is currently no way to manually invalidate your cache, instead you must wait for the Time to Live (`TTL`) to expire on the cached content.
+
+<InlineAlert variant="info" slots="text"/>
+
+`POST` requests are not supported when bringing your own CDN.
+
+## Verifying the caching behavior using response headers
+
+You can verify the caching behavior of GraphQL requests based on the values of the returned response headers when caching is enabled.
+
+**Response headers**
+
+The following response headers are returned when caching is enabled:
+
+- `Age` - On cache `HIT`, cached response age in seconds.
+
+- `Cache-Status` - `HIT` or `MISS`.
+
+- `Date` - Date of the response in UTC.
```

Comment:
Forgot to mention that this header will always be present regardless of caching. While it's a new response header we are introducing, we can remove it from this list.